tests/conftest.py

```diff
@@ -407,6 +408,32 @@ def target_win_tzinfo_legacy(hive_hklm: VirtualHive, target_win: Target) -> Iter
     yield target_win
 
 
+@pytest.fixture
+def target_linux_docker_logs(target_linux: Target, fs_linux: VirtualFilesystem) -> Target:
```

Comment:
Can you move these fixtures to the `test_docker.py` file? I don't think they are currently used in other files.